recipes/gz-sim-yarp-plugins/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set cxx_name = "libgz-sim-yarp-plugins" %}
+{% set plugin_name = "gz-sim-yarp-plugins" %}
+{% set name = "gz-sim-yarp-plugins-split" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/robotology/gz-sim-yarp-plugins/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: bbfda55f551377bff6e4d12abe8ce48897f41e53324d2ddcb24cf6627673676b
+
+build:
+  number: 0
+
+outputs:
+  - name: {{ cxx_name }}
+    script: build_cxx.sh  # [unix]
+    script: bld_cxx.bat  # [win]
+    build:
+      run_exports:
+        - {{ pin_subpackage(cxx_name, max_pin='x') }}
```

Comment:
We are not installing any header, so at the moment no downstream package can actually link our plugins, they can be only loaded at runtime. So it is impossible that any package requires this `run_exports`. For now I guess we can just remove them.

```suggestion
```

Reply:
Clear thanks!